### ISSUE TITLE
DuckDNS feature request public ipv6 update

### ISSUE THREAD

**Issue by udayakumars** [home-assistant/hassio-addons#461]
Scenario: CG-NAT ISP supporting only ipv6 public address with only opened port 80 and 443

Listed below DuckDNS faq
1Q: why can't you detect IPv6 addresses?
1A: our service is hosted in AWS, they do not support IPv6 on Elastic Load Balancers for the account type we have (VPC), when amazon do support it, we will start detecting it.
2Q: Why do I have to specify my IPv6 address, can't you just detect it?
2A: Amazon Web Services (where we host our servers) do not support IPv6 addresses on servers hosted in VPC. VPC is the new/best hosting in AWS, when AWS support IPv6 on VPC Elastic Load Balancers on VPC, then we will start detecting it.

Please could add a optional feature so that the DuckDNS hassio addon could update the hassio public ipv6 instead of DuckDNS detecting it at their server by providing the optional ipv6 in

https://www.duckdns.org/update?domains={YOURVALUE}&token={YOURVALUE}[&ip={YOURVALUE}][&ipv6={YOURVALUE}][&verbose=true][&clear=true]

**Comment by pblculebras** [comment]
I am using IPv6 and I find it frustrating that I have to update the entry manually on DuckDNS.

Can anyone please enable an option that enables refreshing IPv6 addresses?

**Comment by BMarinov** [comment]
I would love for an update or an official workaround. 

@udayakumars @pblculebras 
One option I found is to use the script in this repo:
https://github.com/gtwy/IPv6forDuckDNS

It seems to parse the output of several commands to determine the ipv6 addr. It also calls https://gtwy.net/ip (warning, invalid certificate) to get the ipv4 addr. This URL seems to be outdated, no service providing the IP is running behind it anymore.

The call to the duckdns server can be modified to exclude the IPV4 part and my plan is to resort to the regular way of keeping that one in sync. 

I tried the important bits on my raspi (Debian) and it seems to be doing just what one would expect. Its a bit unfortunate that we have to resort to this approach. But its better than nothing I guess?
